multilint/__init__.py

```diff
@@ -53,8 +60,13 @@ def main():
 
 def run(raw_args):
     settings = load_settings()
+
     args = parser.parse_args(raw_args)
     skip = args.skip or ()
+    paths = list(args.paths)
+
+    if paths:
+        settings['paths'] = paths
```

Comment:
I'd prefer it if we went the other way and set `paths` from `settings['paths']` if `args.paths` is empty, then passed `paths` in below. This way we aren't mutating `settings`.